### ISSUE TITLE
Scammer using your name

### ISSUE THREAD

**Issue by LordWisienka** [davidebbo/davidebbo.github.io#85]
Hi
I just writting to you to inform you, that some Hindi scammer use your name and photo to pretend to be you. You can see this on RinoaPoison on Twitch (https://www.twitch.tv/rinoapoison). She is an scambaiter and messing with scammers. Your alter ego is in stream nr 217, around 1hr55 min of stream.

Kind Regards
Michał Szczepankowski

**Comment by davidebbo** [comment]
Wow, thanks for letting me know! When I go there, I'm not seeing an obvious way to rewind the stream more than 2 minutes. Maybe it's because it's still live, and it will be available for replay later? Sorry, I haven't used twitch much.

**Comment by davidebbo** [comment]
Ok, I found the direct stream link at https://www.twitch.tv/videos/1232393352. Listening for 5 minutes starting at 1h55, and so far I'm not hearing my name.

**Comment by LordWisienka** [comment]
hmm, its current live stream, you may not be able to go back untill is finish. She talking with that guy right now

**Comment by davidebbo** [comment]
Actually, I found it by going to https://www.twitch.tv/videos/1232393352. It's actually at 1h45 that he introduces himself using my name. Not seeing my photo, but maybe that comes later.

**Comment by LordWisienka** [comment]
Ohh, i see you can revind from direct link, it's exacly 2hr00 minutes, when he taping your details.

**Comment by LordWisienka** [comment]
he going on tour microsoft/google stats or something, with poits for help. Its live stram and she talking with him already 30 minutes:P

**Comment by davidebbo** [comment]
Thanks for letting me know!

**Comment by LordWisienka** [comment]
Your welcome. You can do anything with that?:) Just for courious;) What ordinary people can do with that except deleting this page?

**Comment by LordWisienka** [comment]
some guy from stream sent you Tweet about that as well :)

**Comment by davidebbo** [comment]
Fun part is that I updated my MSDN bio just two weeks ago as it still said Microsoft. And clearly that took the scammers by surprise!

**Comment by LordWisienka** [comment]
That was nothing and not as ridiculus as you may think. I don't know if you ever watched content like this, but those calls may be really riddiculus. If you are interested, watch Kitboga on YT or Twitch, he is a master of riddiculus and weird conversations with scammers and those conversations can be really absurd :) Rinoa even spoke once with "Bratt Pitt" and really sometimes it's hard to believe thery are not some actors or arranged conversations:P

**Comment by davidebbo** [comment]
No, I did not even know that scambaiting existed until you pointed me to this one! Not sure I have time to watch a lot of it, but I can see how it can be entertaining.
